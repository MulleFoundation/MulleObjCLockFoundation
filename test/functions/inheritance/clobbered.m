#import <MulleObjC/MulleObjC.h>
#import <MulleObjC/NSDebug.h>

#include "class-system.inc"


int   main( int argc, char *argv[])
{
   [B clobbered];
   return( 0);
}

