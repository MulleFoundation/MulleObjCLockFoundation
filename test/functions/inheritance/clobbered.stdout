+[B(Y) clobbered]
+[B(X) clobbered]
+[B clobbered]
